src/service/owners-impl.js

```diff
@@ -39,7 +67,13 @@ export class Owners {
    * @param {!Element} parentElement
    * @param {!Element|!Array<!Element>} subElements
    */
-  schedulePreload(parentElement, subElements) {}
+  schedulePreload(parentElement, subElements) {
+    this.scheduleLayoutOrPreloadForSubresources_(
+      Resource.forElement(parentElement),
```

Comment:
to reduce unnecessary dependency, use this.resources_.getResourcexxx to access this.